src/js/services.js

```diff
@@ -35,4 +35,7 @@ export const createCourse = (name, length, description) => postData('courses',
 
 export const getSearchData = search => getData(`courses?q=${search}`);
 
+export const deleteCourse = id => deleteData('courses', id)
+  .catch(() => new Error('Server error'));
```

Comment:
what the purpose of this catch?

Reply:
it's purpose to give an error if something goes wrong